nullaway/src/test/java/com/uber/nullaway/jspecify/GenericsTests.java

```diff
@@ -1737,6 +1737,33 @@ public void testRawTypeReceiverCast() {
         .doTest();
   }
 
+  @Test
+  public void testUseOfUnannotatedCode() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.NullMarked;",
+            "import org.jspecify.annotations.NullUnmarked;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "  @NullUnmarked",
+            "  static class nullUnmarkedClass<S> {",
```

Comment:
In Java, class names should start with a capital letter: https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names